src/ApplicationInsights.Kubernetes/ApplicationInsights.Kubernetes.csproj

```diff
@@ -18,7 +18,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="KubernetesClient" Version="[10.0.16, 11.0.0)" />
+    <PackageReference Include="KubernetesClient" Version="12.1.1" />
```

Comment:
This bumps up the lowest version supported.
If I understand correct, the version needs to be uncapped to unblock the bumping up of the dependencies. How about make it `10.0.16`? That would be equivalent to `[10.0.16, )`.

Considerations:
1. Backward compatibility for existing code relying on the lower version;
2. Potentially breaking changes introduced in the K8sClient (major version upgrades);

If that would work for you, could you please update the PR?

Reply:
I upgraded the reference as per your request. Thank you!

Reply:
Cool!